yt_dlp/extractor/drtv.py

```diff
@@ -372,3 +373,107 @@ def _real_extract(self, url):
             'formats': formats,
             'is_live': True,
         }
+
+
+SERIES_API = 'https://production-cdn.dr-massive.com/api/page?device=web_browser&item_detail_expand=all&lang=da&max_list_prefetch=3&path=%s'
+
+
+class DRTVSeasonIE(InfoExtractor):
+    IE_NAME = 'drtv:season'
+    _VALID_URL = r'''(?x)
+                    https?://
+                        (?:
+                            (?:www\.)?(?:dr\.dk|dr-massive\.com)/drtv/(?:saeson)/
+                        )
+                        (?P<id>[\da-z_-]+)
+                    '''
```

Comment:
Why so complicated?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?(?:dr\.dk|dr-massive\.com)/drtv/saeson/(?P<display_id>[\w-]+)_(?P<id>\d+)'
```
also see next comment